Loading pam_limits.so is needed to have limits apply to sessions started with lightdm

--- a/data/pam/lightdm
+++ b/data/pam/lightdm
@@ -18,3 +18,6 @@ password  required pam_unix.so
 # Setup session
 session   required pam_unix.so
 session   optional pam_systemd.so
+
+# Load limits
+session   optional pam_limits.so

--- a/data/pam/lightdm-autologin
+++ b/data/pam/lightdm-autologin
@@ -20,3 +20,6 @@ password  required pam_deny.so
 # Setup session
 session   required pam_unix.so
 session   optional pam_systemd.so
+
+# Load limits
+session   optional pam_limits.so

--- a/data/pam/lightdm-greeter
+++ b/data/pam/lightdm-greeter
@@ -15,3 +15,6 @@ password  required pam_deny.so
 # Setup session
 session   required pam_unix.so
 session   optional pam_systemd.so
+
+# Load limits
+session   optional pam_limits.so
